resources/sdk/webmessagingjava/templates/WebMessagingClient.mustache

```diff
@@ -41,6 +41,27 @@ public class WebMessagingClient {
     private final ArrayList<SessionListener> sessionListeners = new ArrayList<>();
     private ApiClient apiClient;
 
+    private AllowedMedia allowedMediaInbound;
```

Comment:
couldnot find a class for Allowedmedia

Reply:
it will exists once generated using open API file
![image](https://github.com/purecloudlabs/platform-client-sdk-common/assets/14348117/02437789-71ec-4ed7-ab90-a542489d6123)